projects/power_distribution/src/current_measurement.c

```diff
@@ -69,7 +69,10 @@ StatusCode power_distribution_current_measurement_init(PowerDistributionCurrentS
       return status_code(STATUS_CODE_INVALID_ARGS);
     }
 
+    // Add DSEL, EN0, EN1 pins
```

Comment:
Might want to add a TODO here to convert power distro to using the bts7200 + (eventually) bts7040 drivers for toggling the enable pins which will take quite a lot of restructuring, feel free to make a new ticket if there isn't an appropriate one already

Reply:
Sure, created [SOFT-336] for this issue